e2e/browser/e2e.playwright.ts

```diff
@@ -25,7 +25,9 @@ import { LoginFlow } from "./pages/LoginFlow";
 // TODO: Redirected resource tests? I'm not sure what those actually show
 
 test.describe.parallel("Not Logged In", () => {
```

Comment:
I think I've seen tests fail due to hitting the same pod at the same time, maybe it's worth removing the .parallel?

Reply:
Probably so, I'm not sure why it got here in the first place.